assets/js/admin/form-builder.js

```diff
@@ -1595,6 +1636,9 @@
 					EVFPanelBuilder.conditionalLogicAppendFieldIntegration( dragged_el_id );
 					EVFPanelBuilder.paymentFieldAppendToQuantity( dragged_el_id );
 					EVFPanelBuilder.paymentFieldAppendToDropdown( dragged_field_id, field_type );
+
+					// Initializations.
```

Comment:
Date Initializations. 

Notice the previous initialization methods : They're descriptive, like paymentFieldAppendToDropdown. You can just read the method name and realize what is happening without reading a comment. A good code, should be commented. A great code, doesn't need commenting for understanding.

I propose changing the init_datepickers to something like initializeDatepickerWidget, just an example. Then, you can possibly just remove the comment entirely 👍

Reply:
Resolved with e3051942f7b54bebc0ba16ae0399987ede193104